manuscript-manager/components/manuscript/createManuscript.tsx

```diff
@@ -19,25 +20,77 @@ import {
   Button,
 } from "@chakra-ui/react";
 import { ChangeEvent, useState } from "react";
-import { DM_Serif_Display } from "next/font/google";
+import { useEffect } from "react";
+import { handleManuscripts } from "@/utils/handleManuscripts";
 //this component handles the creation of new manuscripts, and will eventually send the manuscript as a request to the backend.
 
-export default function CreateManuscript() {
+interface CreateManuscriptProps {
+  manuscriptToUpdate?: ManuscriptType;
+  setManuscriptsInState: (manuscript: ManuscriptType[]) => void;
+}
+
+export default function CreateManuscript({
+  manuscriptToUpdate,
+  setManuscriptsInState,
+}: CreateManuscriptProps) {
   const [manuscriptID, setManuscriptID] = useState<string>("");
   const [date, setDate] = useState<Date>(new Date());
-  const [wordCount, setWordCount] = useState<number>();
+  const [wordCount, setWordCount] = useState<number | undefined>();
```

Comment:
Test comment: 

Is this so we can have an empty input for the wordcount instead of a 0? I like that, but we will need to ensure the undefined can't make it into the database, as that'll break some math, especially if we're schema-less.